.gitignore

```diff
@@ -0,0 +1,4 @@
+# List files, folders & patterns that git should ignore
+# Remember that no binaries or user-specific files should be tracked
+
```

Comment:
let's put here:
.idea
.vs*
[Dd]ebug/
[Dd]ebugPublic/
[Rr]elease/
[Rr]eleases/
x64/
x86/
build/
bld/
[Bb]in/
[Oo]bj/
[Oo]ut/

Reply:
Intentionally left empty to sum up requests in one file

I would also add

```
# Visual Studio Code
.vscode
# F# package for VS Code
.ionide
# Prefere to compile C++ code by hands with MinGW and usually get executable in the same folder
*.exe
```

Reply:
I would add:
```
# Virtual environment for Python
venv/
```